TESTING.md

```diff
@@ -11,72 +11,82 @@ Recipeezy
 
 # Automated Test Cases:
 
+PRECONDITION TO RUN ALL TESTS:
+1. create a new environment
+2. run `pip install -r requirements.txt` in environment to get requirements
 ---
 Use case name
     Verify login with valid user name and password
 Description
-    Test the Google login page
+    Test for logging in only. This tests for access to the User database and
+    return to the HOME page.
 Pre-conditions
-    User has valid user name and password
+    No preconditions
 Test steps
-    1. Navigate to login page
```

Comment:
This may not be necessary for the professor, but it might not hurt to mention that the virtual environment needs to be activated and that we need to do 'flask run'